maxima/g0/deflate_zsurf/deflate-zsurf.mac

```diff
@@ -0,0 +1,73 @@
+load("modal-basis")$
+load(stringproc)$
+load("out-scripts")$
+
+/* Taking DG fields g_ij and |B| as inputs, this kernel will calculate the other zsurfmetric quantities
+J = det(g_ij))
+b_i = g_iz/sqrt(g_zz)
+*/
+
+deflatezsurf(fh, basisName, polyOrder, cdim) := block(
+/* powerset maxima function will give me combos*/
+
+  disp("basisName", basisName),
+  /* get the full basis*/
+  kill(varsC, varsP, basisC, basisP),
```

Comment:
unnecessary I think, loadBasis calls `kill`, but also less necessary once you populate the list of private vars